karma.conf.js

```diff
@@ -121,36 +120,45 @@ module.exports = function(config) {
         },
         reporters: ['dots'],
         webpack: {
+            mode: 'development',
             devtool: 'inline-source-map',
             externals: {
                 fabric: 'fabric'
             },
             module: {
-                preLoaders: [
+                rules: [
+                    {
+                        test: /\.js$/,
+                        include: /src/,
+                        exclude: /node_modules|bower_components/,
```

Comment:
Doest this use bower?

Reply:
I knew wrong usage, bower did not need to excloud. Thank you.